components/es/burnModal/MKRAmount.tsx

```diff
@@ -34,11 +34,6 @@ const MKRAmountView = ({ setBurnAmount, burnAmount, mkrBalance }: Props) => {
           value={burnAmount.toBigNumber()}
         />
       </Box>
-
-      <Flex mt={3} sx={{ alignItems: 'center', justifyContent: 'flex-start', width: '100%' }}>
-        <Text variant="caps">MKR Balance In Wallet</Text>
-        <Text ml={3}>{mkrBalance ? mkrBalance.toString() : '---'}</Text>
-      </Flex>
     </>
```

Comment:
I'm not sure what this was doing but it's safe to delete?

Reply:
yes, was duplicated because the balance is in the MKRInput